### DISCUSSION TITLE
[Feature Request] Add default setups for new users

### DISCUSSION THREAD

**Discussion by theOehrly** [f1multiviewer/issue-tracker#563] [Feature requests]
This isn't a problem that I personally have had, but it has been relayed to me from a "less technical" user.

They seem to struggle with the concept of setups and with creating and saving custom setups. I think this might be because they're not used to a software that offers many independent windows that can be customized and repositioned freely. Their wish was to have some built-in default setups to get started more easily with multiviewer.

I do suppose that this is not that simple to implement because
1. setups are saved in pixel coordinates, so they'd need to be generated dynamically instead of just shipping a bunch of configs, I suppose
2. we'd need to know what are common setups that people would like

I think that this actually would be nice to have. But it certainly is a "bonus feature".

**Comment by jeppeolesen** [comment]
Marking this as a duplicate of #376
